novawallet/Modules/AssetList/AssetListPresenter.swift

```diff
@@ -22,6 +22,7 @@ final class AssetListPresenter {
     private var hidesZeroBalances: Bool?
     private var shouldShowPolkadotPromotion: Bool = true
     private var hasWalletsUpdates: Bool = false
+    private var selectedChainAssetId: ChainAssetId?
```

Comment:
I don't like this approach. It makes AssetListPresenter much complicated. I would stick with the approach when MainTabBar just pops current details and pushes new one.

Also we don't have here the similar issue we had with governance when referendumId is not available until we fetch it from the chain. Here we already know the chain as we subscribe to it in the AssetDetailsNotificationHandler.